.git-blame-ignore-revs

```diff
@@ -1,2 +1,18 @@
+# The big renaming
+6950b14829d4cc747d9d7d90335ebc0d1fe283fc
+80cdca476750a641c991e1c0a891936bfc5321c5
+387c48ca6e4eda31841b8f58a8361e44137f2592
```

Comment:
I think these are the hashes from the pull request, but not the ones the corresponding commits on master have. :sweat_smile:
```
> git show 387c48ca6e4eda31841b8f58a8361e44137f2592
Schwerwiegend: bad object 387c48ca6e4eda31841b8f58a8361e44137f2592
```
This one should be 0adcc7e8e4212241421500b4419c35faf740a5b1 I think.

Reply:
Aaah... You're right. I'll follow up next week